exporter/datadogexporter/stats.go

```diff
@@ -58,7 +61,7 @@ func computeAPMStats(tracePayload *pb.TracePayload, calculator *sublayerCalculat
 				Weight:   1,
 				TopLevel: true,
 			}
-			statsRawBucket.HandleSpan(weightedSpan, tracePayload.Env, []string{versionAggregationTag}, sublayers)
+			statsRawBucket.HandleSpan(weightedSpan, tracePayload.Env, []string{versionAggregationTag}, emptySublayer)
```

Comment:
Is it equivalent that we pass an empty sublayer here (and thus leave empty the `statsRawBucket` fields) to using the new payload format from the Datadog Agent?

Reply:
😞  This is a good point. Upon closer inspection, no, it is not equivalent. I think due to changes in `RawBucket` and `StatsRawBucket` we're pinned here until the `exportables` package can get updated. I think our payloads would not be handled properly. fml. I'm going to close this...again lol...and followup with the impacted customers and provide some workarounds involving lowering batching time. Ultimately I think we're going to continue encountering issues in Fargate and other resource constrained environments until we both reduce the size of the collector itself (since contrib is large), as well as the resource consumption.